index.d.ts

```diff
@@ -22,6 +22,7 @@ import { EventEmitter } from 'events';
 
 type DummySearchClientV4 = {
   readonly addAlgoliaAgent: (segment: string, version?: string) => void;
```

Comment:
I that case do we still need to keep `addAlgoliaAgent` ?

Reply:
good point, it's indeed not needed anymore
```suggestion
```